eng/targets/Helix.Common.props

```diff
@@ -19,13 +19,12 @@
 
   <!-- queues for helix-matrix.yml pipeline -->
   <ItemGroup Condition="'$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' == 'true' AND '$(_UseHelixOpenQueues)' == 'true' AND '$(IsWindowsOnlyTest)' != 'true'">
-    <HelixAvailableTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Ubuntu.2004.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="OSX.1100.Amd64.Open" Platform="Linux" />
```

Comment:
Why are the two `OSX.*` queues using `Platform="Linux"`❔ We have an `OSX` `@(HelixAvailablePlatform)` item and Libuv.BindTests.csproj attempts to exclude it.

Reply:
Oversight most likely, when things started we mostly just had windows and non windows queues.  There wasn't anything that used to care about linux vs osx